ci_tools/compare_coverage_to_diff.py

```diff
@@ -0,0 +1,161 @@
+""" Functions for comparing coverage output and git diff output
+"""
+import os
+import sys
+import defusedxml.ElementTree as ET
+
+def get_untested_lines(coverage_filename):
+    """
+    Parse a coverage xml file and return a dictionary containing the files and lines
+    which are untested
+
+    Parameters
+    ----------
+    coverage_filename : str
+        The name of the xml file containing the coverage information
+
+    Result
+    ------
```

Comment:
I think the correct section title is `Returns`

Reply:
Fixed 44267e7a